### DISCUSSION TITLE
What user to use for commits in release.yml Github action?

### DISCUSSION THREAD

**Discussion by wimdeblauwe** [jreleaser/jreleaser#1026] [Q&A]
In the example [Github action to release](https://github.com/aalmiray/sb-cli/blob/main/.github/workflows/release.yml), the user `41898282+github-actions[bot]@users.noreply.github.com` is used. Is this a user that any project can/should use? Or is there one per project? If so, where should I get the one for my project?

**Comment by aalmiray** [comment]
This is the default user that matches "GitHub Actions". You may use any other username. However, that matches with the GitHub octocat 😏
